### PR TITLE
FlexOneHARRIER amp-ad support

### PR DESCRIPTION
**Pull request by nikitachang**
Could you merge 'FlexOneHARRIER amp-ad support' into ampproject/amphtml master branch?

We have a Google Corporate CLA.
Our CLA: D.A.Consortium Inc.

Looking forward to hearing from your response!

### CODE REVIEW

**Comment by nikitachang** [comment]
Hi,
I have fixed the errors, may I ask how do I run travis to test it again?
Thank you.

**Comment by nikitachang** [comment]
Hi. I signed the CLA.

**Comment by taisuke-oomi** [comment]
Please proceed with the pull request content.

File: ads/f1h.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  loadScript,
+  validateData,
+} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function f1h(global, data) {
+
```
Comment (alanorozco) [inline ads/f1h.js]: Please remove empty newlines at beginning and end of function body.

File: ads/f1h.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  validateData(data, ['sectionId', 'slot']);
+
+  const scriptUrl = (data['debugsrc'] || ('https:' === window.document.location.protocol ? 'https://' : 'http://') + 'img.ak.impact-ad.jp/fh/f1h_amp.js');
```
Comment (alanorozco) [inline ads/f1h.js]: Is there a particular reason why you can't use HTTPS in all cases?

File: 3p/integration.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/**
+﻿/**
```
Comment (alanorozco) [inline 3p/integration.js]: Please undo this line change.

Comment (nikitachang) [inline 3p/integration.js]: I didn't modify Line 1.  Could you please check what the problem is for me?

Comment (alanorozco) [inline 3p/integration.js]: I'm not sure why it the change was applied by git, but there is definitely something that went wrong with the PR, as other changes appear when they should not. (e.g. the objects named `git` and `cd` with no changes).

Please remove those two empty files. 

As for the first line change, that's probably fine after squash, so don't worry about it.

**Important:** Also note that the Google CLAs need to be signed by all parties in this PR. I won't be able to merget these changes unless @taisuke-oomi signs the agreement, as they made commits as part of this PR.

File: ads/f1h.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function f1h(global, data) {
+  validateData(data, ['sectionId', 'slot']);
+
+  const scriptUrl = data['debugsrc'] || 'https://img.ak.impact-ad.jp/fh/f1h_amp.js';
```
Comment (alanorozco) [inline ads/f1h.js]: Line is over 80 lines. Please split like so:

```javascript
const scriptUrl =
      data['debugsrc'] || 'https://img.ak.impact-ad.jp/fh/f1h_amp.js';
```

**Comment by nikitachang** [comment]
@alanorozco 
Thank you for reviewing.
Please merge files.
I'll test it again after merge, hope our Ads will be running well.  :)

**Pull request merged**